server/src/game-implementations/games/ddr.ts

```diff
@@ -336,9 +358,13 @@ export const DDR_IMPL: GPTServerImplementation<"ddr:DP" | "ddr:SP"> = {
 
 			return DDRFlare.calculate(chart.levelNum, flareLevel);
 		},
+		exScore: (scoreData) => {
```

Comment:
Likewise, I don't think this is a meaningful metric to calculate scores on, because of ex score being mostly a function of Notes In The Chart.